<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно регистрации</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: rgb(255, 231, 156);
	color: rgb(136, 116, 76)
}

QPushButton:hover {
	background: rgb(255, 230, 103);
	color: rgb(77, 76, 53)
}

QPushButton:pressed {
	background: rgb(255, 230, 103);
	color: rgb(77, 76, 53)
}

QLabel{
	background:rgb(255, 167, 123);
	color: rgb(165, 116, 93);
	border: 4px solid rgb(244, 193, 157);
}

QLineEdit{
	background: rgb(255, 238, 222);
	border: 2px solid rgb(255, 213, 188);
	color: rgb(48, 27, 10)
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QLabel" name="T_EnterLogin">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>17</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 213, 175);
color: rgb(165, 116, 93);
border: 4px solid rgb(244, 193, 157)</string>
      </property>
      <property name="text">
       <string>Введите свой логин</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="LE_EnterLogin">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="T_Card">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 213, 175);
color: rgb(165, 116, 93);
border: 4px solid rgb(244, 193, 157)</string>
      </property>
      <property name="text">
       <string>Пин-код от вашей карты</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="B_Reg">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Зарегистрироваться!</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="LE_EnterPassword">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="T_Enter">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="T_EnterPassword">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>17</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 213, 175);
color: rgb(165, 116, 93);
border: 4px solid rgb(244, 193, 157)</string>
      </property>
      <property name="text">
       <string>Введите свой пароль</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LE_RegLogin">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;123123&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="T_RegistrationPassword">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>17</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 213, 175);
color: rgb(165, 116, 93);
border: 4px solid rgb(244, 193, 157)</string>
      </property>
      <property name="text">
       <string>Введите свой пароль</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="T_Registration">
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="LE_Card">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LE_RegPassword">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="18" column="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="13" column="3">
     <widget class="QPushButton" name="B_Enter">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Войти!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="T_RegistrationLogin">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>17</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 213, 175);
color: rgb(165, 116, 93);
border: 4px solid rgb(244, 193, 157)</string>
      </property>
      <property name="text">
       <string>Введите свой логин</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
